data/migrations/20181011153443_quantity_table.js

```diff
@@ -0,0 +1,10 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('quantity', function(tbl) {
+        tbl.increments();
+        tbl.integer('quantity').notNullable();
+     })
+};
```

Comment:
You can include a quantity column on your recipes table instead of making an entirely new table.